api/services/FootprintService.js

```diff
@@ -181,16 +181,26 @@ module.exports = class FootprintService extends Service {
    * @return Promise
    */
   createAssociation(parentModelName, parentId, childAttributeName, values, options) {
-    /*
-     const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
-     const childAttribute = parentModel.associations[childAttributeName]
-     const childModelName = childAttribute.target.name
-
-     values[childAttribute.foreignKey] = parentId
+    const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
```

Comment:
If you don't mind I'll do the review at the same time you push ^^ 
Model presence need to be check like this : 
https://github.com/trailsjs/trailpack-sequelize/blob/master/api/services/FootprintService.js#L26